HermaFx.DataAnnotations/NotDefaultAttribute.cs

```diff
@@ -11,17 +11,21 @@ public class NotDefaultAttribute : ValidationAttribute
 	{
 		private const string _defaultErrorMessage = "The field {0} requires a non-default value";
 
-		public NotDefaultAttribute()
-				: base(string.Format(_defaultErrorMessage))
```

Comment:
This .ctor is mandatory when using an specific ErrorMessage is required. You just need to pass a lambda returning the overriden error message format string.